preamble.tex

```diff
@@ -107,23 +107,23 @@
 \begin{tabular}{rl}
 \resizebox{0.50\textwidth}{!}{
 \begin{tabular}{lr}
-    \multicolumn{2}{l}{\textbf{Soccer League Committee 2023:}}\\
-    Michael Ambrose           & USA (CO-CHAIR)\\
-    Javier E. Delgado Moreno  & Mexico \\
-    Hikaru Sugiura            & Japan\\
-    David Schwarz             & Germany (CHAIR)\\
-    William Plummer           & Australia\\
-    Adrián Matejov            & Slovakia\\
+    \multicolumn{2}{l}{\textbf{Soccer League Committee 2024:}}\\
+    Michael Ambrose           & USA \\
+    Ryely Burtenshaw-Day      & New Zealand \\
+    Ivan Kolarić              & Croatia \\
```

Comment:
@dbscoach it will be interesting to see how LaTeX handles this for sure :slightly_smiling_face:

Reply:
I set up UTF-8 for Adrian already, renders fine now ;)